src/components/FeedPortal/components/TeamCard.jsx

```diff
@@ -10,9 +10,9 @@ const TeamCard = ({ team }) => {
         <InfoComponents team={team} />
       </div>
       <div className="team-card-buttons-container">
-        <Link to={"/project/" + team.project.id + "/workspace"} className="user-btn">Team Workspace</Link>
+        {/* <Link to={"/project/" + team.project.id + "/workspace"} className="user-btn">Team Workspace</Link> */}
         <Link to={"/project/" + team.project.id} className="user-btn">Project Showcase</Link>
-        <Link className="user-btn" to={"/team/checkin/" + team.cohort.id}>Team Standups</Link>
+        <Link className="user-btn" to={"/team/" + team.cohort.id + "/standup"}>Team Standups</Link>
```

Comment:
this one should be the `team.id`
also lets use template strings for injecting variables
\`/team/${team.id}/standup\`